File: code_conversion_js/animal.mjs
export { Animal }
class Animal
{
    name;
    hasFur;
    numberOfLegs;
    sizeInFeet

    constructor(name, hasFur, numberOfLegs, sizeInFeet)
    {
        this.name = name;
        this.hasFur = hasFur;
        this.numberOfLegs = numberOfLegs;
        this.sizeInFeet = sizeInFeet;
    }
    getDescription()
    {
        return `${this.name} has${this.hasFur ? "" : " no"} fur and has ${this.numberOfLegs} legs. It is ${this.sizeInFeet} ft tall.`;
    }
}

File: CodeConversionCS/CodeConversionCS/Animal.cs

public class Animal
{
    public String Name { get; set; }
    public bool HasFur { get; set; }
    public int NumberOfLegs { get; set; }
    public double SizeInFeet { get; set; }

    public Animal(String name, bool hasFur, int numberOfLegs, double sizeInFeet)
    {
        Name = name;
        HasFur = hasFur;
        NumberOfLegs = numberOfLegs;
        SizeInFeet = sizeInFeet;
    }

    public virtual String GetDescription()
    {
        return $"{Name} has" + (HasFur ? "" : " no") + $" fur and has {NumberOfLegs} legs. It is {SizeInFeet} ft tall.";
    }
}